### PR TITLE
View public set test

### PR DESCRIPTION
**Pull request by alansutherland**

### CODE REVIEW

**Comment by alansutherland** [comment]
The test is failing as it seems there's an issue with viewing public sets on the test portal which isn't happening on local or prod [this set](https://portaljs-test.eanadev.org/en/set/66 ) shows me a blank version of the user likes page 
Screenshot 2020-11-09 at 20 41 01

File: nuxt.config.js [hunk 1]
```diff
@@ -246,14 +246,15 @@ if (Number(process.env['ENABLE_XX_USER_AUTH'])) {
     strategies: {
       local: false,
       keycloak: {
-        _scheme: process.env.OAUTH_SCHEME,
+        _scheme: process.env.OAUTH_SCHEME || 'oauth2',
+        origin: process.env.OAUTH_ORIGIN || 'https://auth.europeana.eu',
```
Comment (rwd) [inline nuxt.config.js:250]: This does not appear to be used anywhere. If not, remove it.

File: tests/features/pages/set-page.feature [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Feature: User Set Page
+
+  Scenario: Viewing a public set
+
+    When I open an `user gallery page`
```
Comment (rwd) [inline tests/features/pages/set-page.feature:5]: "a" not "an"

File: tests/features/pages/set-page.feature [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+    Then I see the `user gallery page`
+    And I see a `item count`
```
Comment (rwd) [inline tests/features/pages/set-page.feature:7]: "an" not "a"

File: tests/features/pages/set-page.feature [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+    And I see a `item previews grid`
```
Comment (rwd) [inline tests/features/pages/set-page.feature:8]: "an" not "a"

File: tests/features/pages/set-page.feature [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+    And I see a `item preview`
```
Comment (rwd) [inline tests/features/pages/set-page.feature:9]: "an" not "a"

**Review by rwd — CHANGES_REQUESTED** [review]

File: .github/workflows/support/ci/.env [hunk 6]
```diff
@@ -12,7 +12,13 @@ CTF_SPACE_ID=${CTF_SPACE_ID}
 DISQUS_SHORTNAME=${DISQUS_SHORTNAME}
 ENABLE_LINKS_TO_CLASSIC=1
 ENABLE_SSL_NEGOTIATION=1
+EUROPEANA_API_KEY=${EUROPEANA_RECORD_API_KEY}
 EUROPEANA_ANNOTATION_API_KEY=${EUROPEANA_ANNOTATION_API_KEY}
 EUROPEANA_ENTITY_API_KEY=${EUROPEANA_ENTITY_API_KEY}
-EUROPEANA_RECORD_API_KEY=${EUROPEANA_RECORD_API_KEY}
 EUROPEANA_SET_API_KEY=${EUROPEANA_SET_API_KEY}
```
Comment (lbiedinger) [inline .github/workflows/support/ci/.env:18]: we could also probably get rid of these.

**Review by lbiedinger — APPROVED** [review]

**Review by rwd — APPROVED** [review]

**Pull request merged**